frontend/src/containers/ReportPage/MultichoiceQuestionReport/index.tsx

```diff
@@ -0,0 +1,25 @@
+import React, {FC} from "react";
+import {IQuestionReportMultichoiceData, IQuestionReportRadioData} from "../../../models/report/IReport";
+import PieChartTemplate from "../../../components/ReportTemplates/PieChartTemplate";
+import BarsChartTemplate from "../../../components/ReportTemplates/BarsChartTemplate";
+
+export interface IMultichoiceQuestionReportProps {
+  data: IQuestionReportMultichoiceData;
+}
+
+const MultichoiceQuestionReport: FC<IMultichoiceQuestionReportProps> = ({data}) => {
+
+  const getRandomColor = () => {
```

Comment:
Move it outside of the component itself, a new instance of function will be created on each component render. Also, using regex here is a little bit overkill. Why not just 
```
const hex = Array.from({length: 6})
                       .map(() => (Math.random() * 16) | 0)
                       .toString(16))
                       .join('');
return '#' + hex;//replaceable with string interpolation
```
if you wanted a one-liner